nucleus/common/common-util/src/main/java/com/sun/common/util/logging/LoggingOutputStream.java

```diff
@@ -84,7 +84,7 @@ public class LoggingOutputStream extends ByteArrayOutputStream {
      */
     public LoggingOutputStream(Logger logger, Level level) {
         super();
-        this.logger = logger;
+        this.logger = Logger.getLogger(logger.getName());
```

Comment:
I think we should keep the logger instance, especially as it can be configured and the new logger will not keep this settings.